addons/mrp/models/product.py

```diff
@@ -183,6 +183,24 @@ def _compute_quantities(self):
                 product.outgoing_qty = min(ratios_outgoing_qty) // 1
                 product.free_qty = min(ratios_free_qty) // 1
 
+    def _compute_quantities_dict(self, lot_id, owner_id, package_id, from_date=False, to_date=False):
+        # OVERRIDE
+        bom_kits = [
```

Comment:
you can generate the recordset instead...
`bom_kits = self.filtered(lambda product: self.env['mrp.bom']._bom_find(product=product, bom_type='phantom'))`